qa/suites/upgrade/nautilus-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml

```diff
@@ -20,7 +20,7 @@ upgrade-sequence:
    - sleep:
        duration: 60
    - ceph.restart:
-       daemons: [mon.c]
+       daemons: [mon.c, mgr.x]
```

Comment:
We recently made this change in https://github.com/ceph/ceph/pull/33705 to fix https://tracker.ceph.com/issues/44387. I am afraid reverting it will make the bug reappear.

Reply:
ah, ok. i'll just whitelist the message here too then.  i was hoping to have one test that did the mons before mgr, but i think a manual test for this is sufficient.

Reply:
cool, want to get rid of the qa/suites/upgrade/nautilus-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml change?